rtf_vis_tool/src/Components/CoordinateGrid.js

```diff
@@ -0,0 +1,79 @@
+/* Component which renders the x,y,z axes along with the grid lines along the xy plane. Used within
+Data_Association_PC.js.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Third-Party Package Imports.
+import {Line, Html} from "drei";  // Used to render axes lines and text.
+
+// Create grid lines along the xy plane.
```

Comment:
what are the grid lines for? maybe that can be mentioned in the comment? to help orient the user?